src/home/components/EditTaskForm.vue

```diff
@@ -0,0 +1,317 @@
+          >
+            <span class='mdc-notched-outline'>
+              <span class='mdc-notched-outline__leading'></span>
+
+              <span class='mdc-notched-outline__trailing'></span>
+            </span>
+            <input
+              class='mdc-text-field__input'
+              v-model='title'
+              @change='onChange'
+              type='text'
+              placeholder='Title'
+              aria-label='Title'
+              required
+            />
+          </label>
+        </div>
+        <div class='task-assignee input-span'>
+          <span class='input-label'>Assignee:</span>
+          <label
+            class='assignee-input mdc-text-field mdc-text-field--outlined mdc-text-field--no-label'
+          >
+            <span class='mdc-notched-outline'>
+              <span class='mdc-notched-outline__leading'></span>
+
+              <span class='mdc-notched-outline__trailing'></span>
+            </span>
+            <input
+              class='mdc-text-field__input'
+              v-model='assignee'
+              @change='onChange'
+              type='text'
+              placeholder="Assignee's Name"
+              aria-label='Name'
+              required
+            />
+          </label>
+        </div>
+        <div class='task-date input-span'>
+          <span class='input-label'>Date:</span>
+          <label
+            class='date-input mdc-text-field mdc-text-field--outlined mdc-text-field--no-label'
+          >
+            <span class='mdc-notched-outline'>
+              <span class='mdc-notched-outline__leading'></span>
+
+              <span class='mdc-notched-outline__trailing'></span>
+            </span>
+            <input
+              class='mdc-text-field__input'
+              v-model='date'
+              @change='onChange'
+              type='text'
+              placeholder='DD/MM/YYYY'
+              aria-label='Date'
+              required
+            />
+          </label>
+        </div>
+        <div class='task-dept input-span'>
+          <span class='input-label'>Department:</span>
+
+          <div id='demo-menu' class='mdc-menu-surface--anchor'>
+            <label
+              class='dept-select mdc-text-field mdc-text-field--outlined mdc-text-field--no-label'
+            >
+              <span class='mdc-notched-outline'>
+                <span class='mdc-notched-outline__leading'></span>
+
+                <span class='mdc-notched-outline__trailing'></span>
+              </span>
+              <input
+                @input='toggleDropdown'
+                @click='toggleDropdown'
+                v-model='dept'
+                type='text'
+                class='mdc-text-field__input'
+                placeholder='Select Department...'
+                required
+              />
+            </label>
+
+            <div
+              class='mdc-menu mdc-menu-surface'
+              v-on:MDCMenu:selected.prevent='select'
+            >
```

Comment:
`>` can move up in the above statement. 
Like: 
```
<div
      class='mdc-menu mdc-menu-surface'
      v-on:MDCMenu:selected.prevent='select'>
```

Check other places in the code as well.

Reply:
prettier was auto formatting it, disabled it and now using eslint as default formatter